.tools/transifex/.tx/config

```diff
@@ -47,6 +48,7 @@ trans.pt      = Portuguese.xlf
 trans.pt_BR   = Portuguese (Brazil).xlf
 trans.ro      = Romanian.xlf
 trans.ru      = Russian.xlf
+trans.tr      = Tamil.xlf
```

Comment:
This does not look correct. Should it be "ta"?

Reply:
Yes, it is ta. Looks like a typo.